common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -512,5 +512,14 @@ trait FeatureSwitches {
     sellByDate = never,
     exposeClientSide = true,
   )
+  val PersonaliseSignInAfterCheckout = Switch(
+    SwitchGroup.Feature,
+    "personalise-sign-in-after-checkout",
+    "When ON, the sign in gate shows a personalised message to subscribers and supporters",
```

Comment:
Does the cookie that's used to determine whether to show this persist beyond the initial 'subscribed' session? Just wondering if this should say 'to subscribers & supporters who recently signed up' or similar?

Reply:
We can clarify - "When ON, the sign in gate shows a personalised message to non-signin users, if a GU_CO_COMPLETE (checkout complete) cookie is present containing user type and product information. The cookie expires after two weeks".

More information on this cookie available here: https://github.com/guardian/support-frontend/pull/3743

Reply:
This makes sense, thanks :)